src/engine/strat_engine/filesystem.rs

```diff
@@ -109,6 +156,29 @@ impl Recordable<FilesystemSave> for StratFilesystem {
     }
 }
 
+/// Return total bytes, total bytes used
```

Comment:
This ok?

/// Return total bytes allocated to the filesystem, total bytes used by data/metadata